DEVELOPMENT.md

```diff
@@ -0,0 +1,170 @@
+# Building
```

Comment:
nit: Development? (to match the filename)

Reply:
Done.